### DISCUSSION TITLE
Option to allow serving static files from root

### DISCUSSION THREAD

**Discussion by Isaac-Leonard** [RobinTail/express-zod-api#748] [Q&A] [answered]
Currently static files need to be served from a specific subpath.
Would it be possible to be allowed to serve them from the root path so you could access "localhost:/3000/" and get the index?

**Comment by RobinTail** [comment, marked as answer]
I see at least two ways of achieving that.

1. Using an empty route

```ts
const routing: Routing = {
  "": new ServeStatic(path.join(__dirname, "assets"), {
    // express.static options
  }),
};
```

2. Using a custom express app with a line 

```ts
app.use("/", express.static({ /* ... */ }) );
```

I didn't try them, but probably it would work.
Did you try something like that, @Isaac-Leonard ?

**Comment by RobinTail** [comment]
However, I'd say, that if your express app serves files from the root path, then it is only capable of serving files.
Therefore... I'm sure that it still can be called and API.

I mean... There is no need for routing, IO validations and all that. No need for this library in that case.
